packages/teams-js/test/internal/validOrigins.spec.ts

```diff
@@ -30,7 +30,7 @@ describe('validOrigins', () => {
       expect(result).toBe(true);
     });
     it('validateOrigin returns true if origin for subdomains in teams pre-known allowlist', async () => {
-      const messageOrigin = new URL('https://subdomain.teams.microsoft.com');
```

Comment:
Shouldn't we change these tests to now expect false instead as well, to ensure that this is working as intended?

Reply:
We still allow for wildcard domains for a couple other origins, so it seemed like retrofitting the tests that tested for this case working correctly to use different valid defaults made sense. Does that seem okay?

Reply:
Yup that seems ok to me